HznApplication/HznApp.cpp

```diff
@@ -7,7 +7,7 @@
 std::shared_ptr<Hzn::App> Hzn::createApp()
 {
 	auto app = std::make_shared<HznApp>();
-	/*app->addLayer(new EditorLayer());*/
+	app->addLayer(new EditorLayer("E:\\code file\\groupProject"));
```

Comment:
make this a relative path, store your data in the assets folder and use that path.

Reply:
So will the project folder also be saved in the asset folder？

Reply:
You are passing path to the project directory in the editor layer, why are you doing this by the way?

Reply:
I was hoping to get the local project path before the user opens the editor, and then show the project directory after opening the editor

Reply:
I think since you are using `std::filesystem` it should have some functions in the API that would allow you to retrieve current working directory path, from there on, you can navigate back or edit the path once you get the absolute path to cwd.

Reply:
Okay, sounds great. I will see how to implement it the way you said. Thanks a lot!

Reply:
I think it would be better, to do it that way rather than passing argument to the constructor. Plus, we can work with relative paths in much better way if we do it like that.